qml/Stage/Stage.qml

```diff
@@ -89,18 +93,17 @@ FocusScope {
                 Qt.InvertedLandscapeOrientation;
     }
 
+    Binding {
+        target: topLevelSurfaceList
+        property: "rootFocus"
+        value: interactive
+    }
+
```

Comment:
Are you sure this wont cause a condition where rootFocus is set when stage does not have focus? Since stage will need to have focus before activation of windows, this includes the null window.

Reply:
interactive is cleared whenever the Stage is not in staged, stagedWithSideStage, or windowed state AND allowInteractivity is true: https://github.com/ubports/unity8/pull/239/files#diff-575a1ba4a622df698d6c1dd36cf9f5f1R54

allowInteractivity is set only when the greeter is not active, indicators are closed, notifications are not in 'modal' mode, and the Launcher isn't taking focus: https://github.com/ubports/unity8/pull/239/files#diff-a54a1698b32808661587b74b7e8ac356R326

If interactive is cleared, rootFocus is false.

What case do you see rootFocus being set without the Stage having focus?